package.json

```diff
@@ -206,6 +206,28 @@
             "Appends new memory to bounds of current request, resulting in a growing list."
           ],
           "description": "Behavior when adding more memory beyond the current view."
+        },
+        "memory-inspector.addressRadix": {
+          "type": "number",
+          "enum": [
+            2,
+            8,
+            10,
+            16
+          ],
+          "default": "16",
```

Comment:
```suggestion
          "default": 16,
```

Otherwise VScode gets confused about types and shows 'Default' twice:

![image](https://github.com/eclipse-cdt-cloud/vscode-memory-inspector/assets/62660806/a10812d4-ed75-421b-b0c1-3fcca16205d6)